src/aspire/basis/fle_2d_utils.py

```diff
@@ -87,7 +87,7 @@ def precomp_transform_complex_to_real(ells):
 
     A = sparse.csr_matrix((vals, (idx, jdx)), shape=(count, count), dtype=np.complex128)
 
-    return A
+    return A.conjugate()
```

Comment:
The appears to be re(un)-conjugated in all of its uses. Would it make more sense to remove the conjugation here and above (lines 160 and 170 in fle_2d.py)? Maybe I'm missing a use case.

Reply:
Double check this, thanks.

Reply:
My mistake. I missed that `self.c2r_nus` was being used outside of the case where it is immediately conjugated.

Reply:
No worries, it is possible to do this, but it requires conjugating `betas` and (something else as well I think..)  in the steps that are run over (all) coefs.   Probably this can be revisited one day when it is worth optimizing things further.  Hopefully it's good enough for now.